hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/DistributionSetManagementTest.java

```diff
@@ -252,7 +253,7 @@ private void createAndUpdateDistributionSetWithInvalidVersion(final Distribution
 
         assertThatExceptionOfType(ConstraintViolationException.class)
                 .isThrownBy(() -> distributionSetManagement.create(entityFactory.distributionSet().create().name("a")
-                        .version(RandomStringUtils.randomAlphanumeric(NamedEntity.NAME_MAX_SIZE + 1))))
+                        .version(RandomStringUtils.randomAlphanumeric(NamedVersionedEntity.VERSION_MAX_SIZE + 1))))
                 .as("entity with too long name should not be created");
```

Comment:
could you please fix the assertion error message, `entity with too long version should not be created` instead of `entity with too long name should not be created`. It's in multiple places in this file